addons/osio-addon/src/main/java/io/fabric8/launcher/osio/OsioEmptyLaunchMissionControl.java

```diff
@@ -0,0 +1,88 @@
+package io.fabric8.launcher.osio;
+
+import javax.enterprise.context.Dependent;
+import javax.inject.Inject;
+
+import io.fabric8.launcher.core.api.Boom;
+import io.fabric8.launcher.core.api.ImmutableBoom;
+import io.fabric8.launcher.core.api.MissionControl;
+import io.fabric8.launcher.core.api.Projectile;
+import io.fabric8.launcher.core.api.events.StatusMessageEventBroker;
+import io.fabric8.launcher.core.api.projectiles.CreateProjectile;
+import io.fabric8.launcher.core.api.projectiles.context.CreateProjectileContext;
+import io.fabric8.launcher.osio.client.OsioWitClient;
+import io.fabric8.launcher.osio.client.Space;
+import io.fabric8.launcher.osio.projectiles.ImmutableOsioLaunchProjectile;
+import io.fabric8.launcher.osio.projectiles.OsioLaunchProjectile;
+import io.fabric8.launcher.osio.projectiles.context.OsioProjectileContext;
+import io.fabric8.launcher.osio.steps.GitSteps;
+import io.fabric8.launcher.osio.steps.OpenShiftSteps;
+import io.fabric8.launcher.osio.steps.WitSteps;
+import io.fabric8.launcher.service.git.api.GitRepository;
+import io.fabric8.launcher.service.openshift.api.ImmutableOpenShiftProject;
+import io.fabric8.openshift.api.model.BuildConfig;
+
+
+@Dependent
+public class OsioEmptyLaunchMissionControl implements MissionControl<OsioProjectileContext, OsioLaunchProjectile> {
```

Comment:
How is this different from `OsioLaunchMissionControl`?

Reply:
The only difference is that we are not pushing anything to Github repo.